docs/use-cases.rst

```diff
@@ -3,6 +3,23 @@ Typical Use-Cases
 
 .. START Introduction
 
+The use of Fast RTPS is highly varied, allowing a large number of configurations depending on the scenario in which
+this library is applied. That is why this section defines the implementations for the typical use cases
+presented when using the RTPS protocol for communications.
+
++ :ref:`use-case-fast-rtps-over-wifi`. Presents the case for using Fast RTPS in scenarios where discovery through
+  multicast communication is a challenge. To address this problem, the use of the initial peers list by which the
+  address-port pairs of the remote participants are defined is presented (See :ref:`use-case-initial-peers`).
+  It also specifies how to disable the
+  multicast discovery mechanism (See :ref:`use-case-disabling-multicast-discovery`).
++ :ref:`wide_deployments`. Describes the recommended configurations for using Fast RTPS in environments with a high
+  number of deployed communicating agents. These are the use of a centralized server for the discovery phases
+  (See :ref:`server-client-discovery-use-case`), and the Fast RTPS STATIC discovery mechanism for well known network
+  topologies (See :ref:`wide_deployments_static`).
++ :ref:`fastrtps_ros2`. As the default middleware implementation in the
+  `OSRF <https://www.openrobotics.org/>`_ `ROS 2 <https://index.ros.org/doc/ros2/>`_, this tutorial is an explanation
+  of how to take full advantage for Fast-RTPS wide set of capabilities in a ROS 2 project.
```

Comment:
```suggestion
+ :ref:`fastrtps_ros2`. Since Fast-RTPS is the default middleware implementation in the
  `OSRF <https://www.openrobotics.org/>`_ `Robot Operation System 2 (ROS 2) <https://index.ros.org/doc/ros2/>`_,
  this tutorial is an explanation of how to take full advantage for Fast-RTPS wide set of capabilities in a ROS 2 project.
```